baseclass/README.md

```diff
@@ -0,0 +1,50 @@
+# Apigee Java Callouts Base Class
+
+This class contains common logic for use in creating Apigee Java Callouts.
+
+For more information about Java Callouts, visit: https://docs.apigee.com/api-platform/reference/policies/java-callout-policy
+
+## Usage
+
+This github repository contains sample Java callout source code which shows how this base class can be used.
```

Comment:
Nit, but this repository might contain more than just the base class and its samples. Could we update this line to reference only the directories that are relevant?